### PR TITLE
working post...almost working option 2

### PR DESCRIPTION
**Pull request by ponchieponcho**
@NickWilsonDev  👀 please

### CODE REVIEW

File: PhonebookApp_webserver.js [hunk 1]
```diff
@@ -12,74 +12,26 @@ const rl = readline.createInterface({
...
-//     } else if (request.method === "PUT") {
-//         console.log('Put')
-//     } else if (request.method === "DELETE") {
-//         console.log('Delete')
-//     }
-
-// }).listen(5000);
-
 var readContacts = function (callback) {
     fs.readFile(pbFileName, (err, data)=> {
```
Comment (NickWilsonDev) [inline PhonebookApp_webserver.js]: I like the arrow function

File: PhonebookApp_webserver.js [hunk 2]
```diff
@@ -12,74 +12,26 @@ const rl = readline.createInterface({
...
+            fs.writeFile(pbFileName, JSON.stringify(data), function (err) {
             if (err) throw err;       
             })
         })
 }
 
 var server = http.createServer(function (request, response) {
     console.log(request.method, request.url)
+
     if (request.method === "GET") {
```
Comment (NickWilsonDev) [inline PhonebookApp_webserver.js]: Looking good, I like to think for overall strategy, GET request goes with listing or displaying your contacts, POST request for adding new contacts, PUT request for updating/changing a particular contact, DELETE request for deleting/removing a contact, that gives you the CRUD features create read update delete.

File: PhonebookApp_webserver.js [hunk 3]
```diff
@@ -136,8 +88,15 @@ var mainMenu = function () {
         else if (answer === '2') {
             rl.question('Name: ', function (name) {
```
Comment (NickWilsonDev) [inline PhonebookApp_webserver.js:147]: You may not need the rl.questions, depending on how he has us continue this project.  The front end side may have a form that will gather the data like name number ect, and then pass it to our backend in JSON format.

File: PhonebookApp_webserver.js [hunk 4]
```diff
@@ -136,8 +88,15 @@ var mainMenu = function () {
...
                 rl.question('Phone Number: ', function (phone) {
-                    phoneBook.push({ 'firstName': name, 'phoneNumber': phone });
-                    console.log('Entry stored for ' + name);
+                    // phoneBook.push({ 'firstName': name, 'phoneNumber': phone });
+                    // console.log('Entry stored for ' + name);
+                    let newContact = {'firstName': name, 'phoneNumber': phone }
```
Comment (NickWilsonDev) [inline PhonebookApp_webserver.js:151]: liking the let variable declaration

**Review by NickWilsonDev — COMMENTED** [review]
Looking good, keep it up

File: PhonebookApp_webserver.js (hunk 2 above)
Comment (dbaile6) [reply PhonebookApp_webserver.js]: Likewise, that's a good way to remember it.

**Pull request merged**